src/components/footer/footer.module.scss

```diff
@@ -0,0 +1,50 @@
+.root {
+    padding: 50px 2% 30px;
+    position: fixed;
```

Comment:
It should not be fixed. To make it stick to the bottom use flexbox.

Reply:
Oh, I got the issue